context-config-store/store/persistence/config_persistence.go

```diff
@@ -0,0 +1,16 @@
+package persistence
+
+import (
+	api "github.com/kurtosis-tech/kurtosis/context-config-store/api/golang"
+)
+
+type ConfigPersistence interface {
+	// init initializes the persistence. It does nothing it's already initialized
```

Comment:
sensing a typo here It does nothing it's already initialized

Reply:
I think I might be alittle dyslexic when it comes to if/it/is